.github/workflows/continuous-integration.yml

```
@@ -109,18 +109,21 @@ jobs:
         if: ${{ !cancelled() && hashFiles(format('{0}/composer.json', inputs.plugin-key)) != '' }}
         run: |
           echo -e "\033[0;33mChecking for common dependencies between plugin and core...\033[0m"
-          PLUGIN_DEPS=$(composer show --direct --format=json | jq -r '.installed[] | .name' | grep -v '^glpi-project/tools$')
+          PLUGIN_DEPS=$(composer show --direct --format=json | jq -r '.installed[] | .name')
           CORE_DEPS=$(cd ../../ && composer show --direct --format=json | jq -r '.installed[] | .name')
-          HAS_MATCH=false
+          HAS_MATCH='false'
           for plugin_dep in $PLUGIN_DEPS; do
             for core_dep in $CORE_DEPS; do
-              if [ "$plugin_dep" == "$core_dep" ]; then
+              if [ "$plugin_dep" == "$core_dep" ] && [ "$plugin_dep" != "glpi-project/tools" ]; then
```

We use the double bracket syntax is used everywhere in this file, we could use it here too.
Also, it seems to be safer, see https://mywiki.wooledge.org/BashFAQ/031.

```suggestion
              if [[ "$plugin_dep" == "$core_dep" && "$plugin_dep" != "glpi-project/tools" ]]; then
```